app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+    class RockAndPebbleType < Types::BaseObject
+        field :id, ID, null: false
+        field :rock, Types::UserType, null: false
```

Comment:
I'm not positive but I think this might need to be null: true. Because it is technically on the user class
and not always going to be active on a user...but I'm not positive.

Reply:
Gotcha - I'll go back in a bit and dig into graphql's null again. Thanks for catching this.